models/doctype/SalesInvoice/Templates/Minimal.vue

```diff
@@ -59,7 +59,7 @@
         <div
           class="uppercase text-sm font-semibold tracking-widest text-gray-800 ml-8"
         >
-          From
+          {{ doc.doctype === 'SalesInvoice' ? 'From' : 'To' }}
```

Comment:
@piyushsinghania this is a good solution but can we make this a computed propery in the base template? Something like `isSalesInvoice` and then use that in all the templates. That would make it better.

Reply:
@ankitsinghaniyaz  changes are done. Computed property isSalesInvoice created and minimal template is refactored. Going for refactoring other templates.